BuildingBlocks/src/BuildingBlocks.Application/MediatR/QuotaEnforcerBehavior.cs

```diff
@@ -9,16 +9,21 @@ namespace Enmeshed.BuildingBlocks.Application.MediatR;
 public class QuotaEnforcerBehavior<TRequest, TResponse> : IPipelineBehavior<TRequest, TResponse> where TRequest : notnull
 {
     private readonly IMetricStatusesRepository _metricStatusesRepository;
-    private readonly IUserContext _userContext;
+    private readonly IUserContext? _userContext;
 
-    public QuotaEnforcerBehavior(IMetricStatusesRepository metricStatusesRepositories, IUserContext userContext)
+    public QuotaEnforcerBehavior(IMetricStatusesRepository metricStatusesRepositories, IEnumerable<IUserContext> userContexts)
```

Comment:
> Now you can register the `QuotaCheckerImpl` in case of the ConsumerApi.

How should I do this? Or better, how should I prevent this from happening on the AdminAPI? Should I remove the registering from the Quotas module and add it to the ConsumerAPI `Program.cs` file itself?

Reply:
> Should I remove the registering from the Quotas module and add it to the ConsumerAPI Program.cs file itself?
Currently there is no such registering. But yes, the registering should be in Program.cs. Since it isn't Quota-specific, it wouldn't make sense to have it in the Quotas module.